app/services/discovery_report.rb

```diff
@@ -41,12 +52,11 @@ def process_dobj(dobj)
       errors[:files_found_mismatch] = true unless counts[:files_in_manifest] == counts[:files_found]
     end
 
-    errors[:empty_files] = true if counts[:empty_files] > 0
     errors[:empty_object] = true if counts[:total_size] > 0
```

Comment:
not touched in this PR, so won't hold up merge over it, but this seems backwards to me?  i.e., should be something like `errors[:empty_object] = true unless counts[:total_size] > 0` or `errors[:empty_object] = true if counts[:total_size] == 0`?  or am i misunderstanding/missing something?

Reply:
#306